### PR TITLE
Various docs improvements

### PR DESCRIPTION
**Pull request by sclu1034**
Minor improvements and fixes in the API docs.

### CODE REVIEW

**Comment by Aire-One** [comment]
Fix #3306

**Review by Aire-One — APPROVED** [review]
Thanks, this is the kind of rewordings/improvement that help newcomers.

**Review by Elv13 — APPROVED** [review]

**Comment by Elv13** [comment]
@Aire-One Hi, I sent you an invite to become an "official" approver. Can you accept it, dismiss this review, then approve again so the merge bot kicks in?

Thanks

**Review by Aire-One — APPROVED** [review]

**Comment by Aire-One** [comment]
> @Aire-One Hi, I sent you an invite to become an "official" approver. Can you accept it, dismiss this review, then approve again so the merge bot kicks in?
> 
> Thanks

Oh, it's embarrassing... It seems GitHub only send notification for joining an organization via email, and I missed it because of my filter over GitHub emails that I basically never check since we have the notification dashboard on site...

Anyway, I'm in now! :smile:

File: docs/07-my-first-awesome.md
Comment (actionless) [inline docs/07-my-first-awesome.md:69]: btw we need to add `gtk` one here

**Pull request merged**
